torch_geometric/explain/explanations.py

```diff
@@ -0,0 +1,124 @@
+"""Class representing explanations."""
+from typing import Optional, Tuple, Union
+
+from torch import Tensor
+
+from torch_geometric.data import Data
+
+
+class Explanation(Data):
```

Comment:
Not totally convinced yet that this needs to be a `Data` object. Any reason for that? What is the functionality form `Data` we need here? Do we need it to create subgraphs from the original graph?

Reply:
For creating visualization or downstream tasks based on explanations, we might need methods to create subgraphs. Also, the explanation is a graph with specific attributes, I thought it would be good to reuse the methods that can be already in `pyg` for handling this kind of data.